.github/workflows/main.yml

```diff
@@ -3,146 +3,69 @@ on:
-            INDEX=${{ matrix.ocaml-compiler }}
-            OCAML_VERSION=`python .github/workflows/ocaml_versions.py $INDEX`
-            echo "OCAML_VERSION=$OCAML_VERSION" \
-            >> $GITHUB_ENV
-            echo "OCAML_VERSION = $OCAML_VERSION"
-      - uses: ocaml/setup-ocaml@v2
-        with:
-          ocaml-compiler: ${{ env.OCAML_VERSION }}
-      ## The TLAPM test suite requires the kaputt package.
-      - name: Install OCaml dependencies
-        run: opam install kaputt
-      - uses: actions/cache@v2
-        id: cache
-        with:
-          path: ${{ matrix.DOWNLOADS }}
-          key: ${{ matrix.DOWNLOADS }}
-      ## Create the installer and run the 'fast' test suite as smoke tests before
-      ## installers are uploaded to the Github release.
-      - name: Build and Test TLAPM installer
-        run: |
-            eval $(opam env)
-            ./configure
-            cd tools/installer
-            ./tlaps-release.sh
-            ./${{ matrix.INSTALLER }} -d ../..
-            cd ../..
-            PATH=$(pwd)/bin:$(pwd)/lib/tlaps/bin:$PATH make test
-      - name: Upload Release Asset
-        if: matrix.ocaml-compiler == '2'
-        id: upload-release-asset
-        uses: actions/upload-release-asset@v1.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.get_release_info.outputs.upload_url }}
-          asset_path: tools/installer/${{ matrix.INSTALLER }}
-          asset_name: ${{ matrix.INSTALLER }}
-          asset_content_type: application/octet-stream
-      ## Run the full test suite.
-      - name: Download and Test TLAPM installer
-        if: >-
-            matrix.operating-system == 'ubuntu-latest' &&
-            matrix.ocaml-compiler == '2'
-        ## The test suite has known failures, thus keep going until the
-        ## test suite has been fixed (TODO).
-        continue-on-error: true
-        run: |
-            ls -lah
-            eval $(opam env)
-            ocaml --version
-            make
-            PATH=$(pwd)/bin:$(pwd)/lib/tlaps/bin:$PATH which tlapm
-            PATH=$(pwd)/bin:$(pwd)/lib/tlaps/bin:$PATH which zenon
-            PATH=$(pwd)/bin:$(pwd)/lib/tlaps/bin:$PATH tlapm --version
-            PATH=$(pwd)/bin:$(pwd)/lib/tlaps/bin:$PATH make testall
-      - name: Print Test Results
-        if: matrix.operating-system == 'ubuntu-latest'
-        run: cat test/tests.log
+    test:
+        name: Build TLAPS installer and test it
+        runs-on: ${{ matrix.operating-system }}
+        strategy:
+            matrix:
+                operating-system: [
+                    macos-latest,
+                    ubuntu-latest]
+                ocaml-compiler: [
+                    '0',
+                    '1',
+                    '2',
+                    ]
+                include:
+                    - operating-system: macos-latest
+                      INSTALLER: tlaps-1.5.0-i386-darwin-inst.bin
```

Comment:
The TLAPS version number appears in several places in this file, in `pr.yml` and in `release.yml`. Is there a way to factor it out, so we don't have to change so many numbers when preparing a new release?